newsroom/__init__.py

```diff
@@ -19,8 +19,8 @@
 ELASTIC_PREFIX = "CONTENTAPI_ELASTICSEARCH"
 
 SCHEMA_VERSIONS = {
-    "wire": 3,
-    "agenda": 5,
+    "wire": 4,
+    "agenda": 6,
```

Comment:
not sure if we should bump it, that would trigger reindexing automatically but it's rather an optimization than something that would be needed, so would keep those as it is

Reply:
what do you think @MarkLark86 ?

Reply:
I agree. We should only bump these versions if the system breaks without a re-index.